core/src/saros/concurrent/management/HeartbeatDispatcher.java

```diff
@@ -48,6 +49,7 @@ public void start() {
 
   private void dispatchHeartbeats() {
     // clientDocs should only be accessed by the main thread
+    // FIXME syncExec not asyncExec
```

Comment:
Could you add a reason? I think it is hard to verify whether an old "FIXME"s without a reason is still relevant.